src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working.");
```

Comment:
Write more informative messages in methods. The message should indicate what type of `Machine` has stopped working. For example: `Bulldozer has stopped its work.`